paper-menu-button.html

```diff
@@ -1,80 +1,265 @@
+<link rel="import" href="../paper-material/paper-material.html">
+<link rel="import" href="../neon-animation/animations/fade-in-animation.html">
+<link rel="import" href="../neon-animation/animations/fade-out-animation.html">
+<link rel="import" href="paper-menu-button-animations.html">
+
 <!--
+`paper-menu-button` allows one to compose a designated "trigger" element with
+another element that represents "content", to create a dropdown menu that
+displays the "content" when the "trigger" is clicked.
 
-`paper-menu-button` works together with a button and a `paper-dropdown` to
-implement a button that displays a drop-down when tapped on.
+The child element with the class `dropdown-trigger` will be used as the
+"trigger" element. The child element with the class `dropdown-content` will be
+used as the "content" element.
 
-The child element with the class `dropdown` will be used as the drop-down
-menu. It should be a `paper-dropdown` or other overlay element.
+The `paper-menu-button` is sensitive to its content's `iron-select` events. If
+the "content" element triggers an `iron-select` event, the `paper-menu-button`
+will close automatically.
 
 Example:
 
     <paper-menu-button>
-        <paper-icon-button icon="menu" noink></paper-icon-button>
-        <paper-dropdown class="dropdown">
-            <core-menu class="menu">
-                <paper-item>Share</paper-item>
-                <paper-item>Settings</paper-item>
-                <paper-item>Help</paper-item>
-            </core-menu>
-        </paper-dropdown>
+      <paper-icon-button icon="menu" class="dropdown-trigger"></paper-icon-button>
+      <paper-menu class="dropdown-content">
+        <paper-item>Share</paper-item>
+        <paper-item>Settings</paper-item>
+        <paper-item>Help</paper-item>
+      </paper-menu>
     </paper-menu-button>
 
-@group Paper Elements
-@element paper-menu-button
-@status unstable
+### Styling
+
+The following custom properties and mixins are also available for styling:
+
+Custom property | Description | Default
+----------------|-------------|----------
+`--paper-menu-button-dropdown-background` | Background color of the paper-menu-button dropdown | `#fff`
+`--paper-menu-button` | Mixin applied to the paper-menu-button | `{}`
+`--paper-menu-button-dropdown` | Mixin applied to the paper-menu-button dropdown | `{}`
+
+
 @hero hero.svg
 @demo demo/index.html
 -->
-<link href="../polymer/polymer.html" rel="import">
-<link href="../core-a11y-keys/core-a11y-keys.html" rel="import">
-<link href="../core-dropdown/core-dropdown-base.html" rel="import">
-
-<polymer-element name="paper-menu-button" extends="core-dropdown-base" relative>
-<template>
 
+<dom-module id="paper-menu-button">
   <style>
     :host {
       display: inline-block;
+      position: relative;
+      padding: 8px;
+      outline: none;
+      user-select: none;
+      -webkit-user-select: none;
+      cursor: pointer;
```

Comment:
Does it need to set the `user-select` and `cursor` properties here?  I think whatever it is set on the `dropdown-trigger` (e.g. paper-icon-button) that should be good enough.